### ISSUE TITLE
Local Templating and settings is not working

### ISSUE THREAD

**Issue by Orrison** [marcellino-ornelas/create-react-component#5]
I have followed the instructions in the readme and it seems that my local settings and templates are not taking hold. For example here are my settings:
```
{
	"test": false,
	"css": true,
	"index": true,
	"default": true,
	"cssType": "module.css",
	"templates": true,
	"extendCwd": "./src",
	"verbose": false
}
```
Yet when I run a command to make a component, such as:
```
ccr create Wow
```

None of my settings take affect, for example the css file just comes out as Wow.css even though template dictates it should be named `'./{{= it.component.name }}.{{= it.settings.cssType}}'` so I would expect the file to come out as `Wow.module.css`

Am I doing something wrong?

**Comment by marcellino-ornelas** [comment]
Hey @Orrison. Thank you for reporting this bug! You were right the `cssType` setting wasn't working properly. I fixed this in version 2.1.4. you can update from npm. 

I checked all other settings they seem to be working fine for me. If other settings are failing for you, just let me know which ones so I can fix them for you! Thanks

**Comment by Orrison** [comment]
Looks like it is working now, thank you!
